.env.example

```
@@ -0,0 +1,6 @@
+MOCHA_BAIL=false
+
+RPC_URL='http://127.0.0.1:8545'
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)